docs/application/dotnet/samples/ui/overview.md

```diff
@@ -195,20 +195,38 @@ for TD:
 	<tbody>
```

Comment:
Would you please check ui/overview.md file whether it is overwritten or not?
It looks like the fix is reverted.
 - The alphabetical order is changed.
 - The typo fix is reverted (Animaton -> Animation)

Reply:
- The alphabetical order :  Non-NUI samples are listed first and then NUI samples are listed.
- Typo: Master branch has the typo. When master branch was copied to this branch, the typo was added. It was fixed.